apps/gitness/src/pages-v2/repo/repo-summary.tsx

```diff
@@ -50,7 +50,7 @@ export default function RepoSummaryPage() {
   const [gitRef, setGitRef] = useState<string>('')
   const [currBranchDivergence, setCurrBranchDivergence] = useState<CommitDivergenceType>({ ahead: 0, behind: 0 })
   const [branchTagQuery, setBranchTagQuery] = useState('')
-  const manageTokensLink = '/settings/keys'
+  const manageTokensLink = '/profile-settings/keys'
```

Comment:
should this be updated with new routes function?

Reply:
yes this should be added to route definitions. I can help with adoption.

Reply:
fixed